src/utils/customErrors.ts

```
@@ -5,3 +5,7 @@ export class ContentTypeError extends Data.TaggedError("ContentTypeError")<
 > {}
 
 export class JsonError extends Data.TaggedError("JsonError")<Readonly<{}>> {}
+
+export class SlackUserRetrieveError extends Data.TaggedError(
+  "SlackUserRetrieveError"
+)<Readonly<{}>> {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)